modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -18240,7 +18240,7 @@ thousands-delimiter=,
 thousands-separator=Thousands Separator
 thread=Thread
 thread-as-question-by-default=Thread as Question by Default
-thread-cannot-be-deleted.-anonymize-instead=Thread cannot be deleted. Anonymize instead.
+thread-cannot-be-deleted.-anonymize-it-instead=Thread cannot be deleted. Anonymize it instead.
```

Comment:
Can you move this change outside of the `build lang` commit? This is not autogenerated.